tools/packaging/common/envoy_bootstrap.json

```diff
@@ -271,13 +276,20 @@
           }
         },
         {{- end }}
+        {{- end }}
         "load_assignment": {
           "cluster_name": "xds-grpc",
           "endpoints": [{
             "lb_endpoints": [{
               "endpoint": {
                 "address":{
+                  {{- if not .proxy_via_agent }}
                   "socket_address": {{ .pilot_grpc_address }}
+                  {{- else }}
+                  "pipe": {
+                    "path": "./etc/istio/proxy/XDS"
+                  }
+                  {{- end }}
```

Comment:
nit: in future we can remove `max_requests_per_connection` if using proxy. No need to make a new connection